Azure/sonic-buildimage
# [docker-config-engine-stretch]: Fix dependency typo PYTHON2_SWSSCOMMON

This commit fixes a typo in the fix delivered in PR #6538



**- Why I did it**
syncd fails on the armhf platform within sonic-config-engine/portconfig.py when importing the following
    'from swsscommon.swsscommon import ConfigDBConnector'

**- How I did it**
Corrected a typo in the sonic-config-engine-stretch makefile dependencies section
 
**- How to verify it**
Run the new image on the armhf platform and verify that 'show interfaces status' shows the expected results

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
